WhyTLA+.md

```diff
@@ -0,0 +1,25 @@
+# Why Support TLA+
+
+### What TLA+ Does
+
+One of the costliest elements of software development — second only to finding out what to build — is ensuring the system does what it is meant to do. That is why developers spend much of their time on quality assurance through code reviews, testing (functional, performance, security), and a great deal of bug-fixing. Some bugs are costlier than others because:
+
+- They are difficult to reproduce during the development phase (**discovery**).
+- They are *design* flaws that require significant rework to fix (**correction**).
+- They cause significant damage, such as downtime, data loss, or security vulnerabilities (**impact**).
+
+Because they are non-deterministic and complex, concurrent and distributed systems, tend to suffer from bugs that are costly for all those reasons.
+
+TLA+ is a specification language and a set of tools that offer a cost-effective way of finding and fixing flaws early in development when those flaws are cheaper to fix. Users write a precise and concise description of the system and their expectations from its behavior in TLA+, and the tools then check whether the expectations hold and, if not, how they might break.
+
+### Why TLA+
+
+TLA+ is particularly attractive for industry use compared to other formal methods. Its unique strengths in specifying concurrent and distributed systems make it applicable precisely where mainstream software needs help the most. It can specify arbitrary properties of systems of arbitrary complexity and "ordinary" developers can still learn it in a few weeks. Automated verification with a model checker makes the cost of verification low enough to compare favorably with traditional assurance methods.
+
+For those reasons, TLA+ is among the most popular formal methods in industry, employed successfully by Amazon, Oracle, Microsoft, ElasticSearch, MongoDB, and others. This [technical report from Amazon](http://lamport.azurewebsites.net/tla/amazon-excerpt.html) relates how engineers learned TLA+ in just a few week. They then found and fixed potentially catastrophic and hard-to-reproduce problems in several AWS services.
```

Comment:
```suggestion
For those reasons, TLA+ is among the most popular formal methods in industry, employed successfully by Amazon, Oracle, Microsoft, ElasticSearch, MongoDB, CrowdStrike, Informal Systems, and others. This [technical report from Amazon](http://lamport.azurewebsites.net/tla/amazon-excerpt.html) relates how engineers learned TLA+ in just a few week. They then found and fixed potentially catastrophic and hard-to-reproduce problems in several AWS services.
```

Reply:
CrowdStrike -- sure. But we should only specifically list companies worth over $1B and have a Wikipedia entry (or, at least, are so famous that everyone has heard of).

Reply:
I thought you wanted to have a proof of TLA+ being used in industry. I only have seen TLA+ specs publicly available from Microsoft, ElasticSearch, MongoDB, and Informal Systems. Amazon's specs are hidden under NDA. Oracle? Never seen anything.

Do we all agree about this criterion: But we should only specifically list companies worth over $1B and have a Wikipedia entry (or, at least, are so famous that everyone has heard of)?

Reply:
Docker can be [added](https://github.com/docker/swarmkit/blob/master/design/tla/WorkerSpec.tla)

Reply:
To keep the paragraph short, why don't we lobby our peers to have their employers/companies added to https://github.com/tlaplus/awesome-tlaplus#users and link to it here? If say, an Oracle employee, created the PR that added Oracle to https://github.com/tlaplus/awesome-tlaplus#users, I'd assume this to be good enough to convince execs.